static/main.css

```diff
@@ -177,3 +177,18 @@ ul li button {
   position: static;
   margin-right: 8px;
 }
+
+.InputTransferKey {
+  display: grid;
+  grid-template-columns: 100px 10px 100px 10px 100px 10px;
+  grid-gap: 10px;
+  padding-bottom: 10px;
+}
+
+.InputTransferKey .seperator {
+  text-align: center;
+}
+
+.InputTransferKey .seperator::after {
+  content: '-';
```

Comment:
instead can we use `-` in the `div`? I find this `content` property is only really useful in cases where you don't have access to generating the dom directly, but in this case we do